### ISSUE TITLE
Update for SIMD stabilization?

### ISSUE THREAD

**Issue by egilburg** [TeXitoi/benchmarksgame-rs#67]
Is this the repo that's currently being used for https://benchmarksgame-team.pages.debian.net/benchmarksgame/faster/rust.html ? If so, are there planned updates given the recent SIMD stabilization?

**Comment by llogiq** [comment]
Yes, this is (mostly) the repo. We haven't yet found the time to write up SIMD versions of the benchmarks yet, though. Feel free if you want to help.

**Comment by llogiq** [comment]
Update: My fannkuch_redux SIMD implementation was thrown out (to avoid a SIMD arms race on that benchmark). I have an idea for a different non-SIMD implementation, though. I also have an SIMD spectralnorm implementation, although the current autovectorized one I recently built is already plenty fast.

I may try n_body (after failing with autovectorization because of differences between CPUs) again later, perhaps Mandelbrot, although I see little gains to be made.

**Comment by indo-dev-0** [comment]
> My fannkuch_redux SIMD implementation was thrown out (to avoid a SIMD arms race on that benchmark). 

Are there any details on this?

**Comment by llogiq** [comment]
See [benchmarksgame issue #44](https://salsa.debian.org/benchmarksgame-team/benchmarksgame/-/issues/44).
